dev-docs/feature-responsive.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+title: Responsive Ad Sizes - <font color="red">NEW!</font>
```

Comment:
Let me know if I'm misunderstanding, but I think this page's content basically already exists [in the `sizeConfig`/ `labels` section of the API reference](http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Configure-Responsive-Ads).

I think instead of adding this page we should link users there.